crates/lib/src/cli.rs

```diff
@@ -100,6 +100,13 @@ pub(crate) struct UpgradeOpts {
     #[clap(long = "soft-reboot", conflicts_with = "check")]
     pub(crate) soft_reboot: Option<SoftRebootMode>,
 
+    /// Download and stage the update without applying it.
+    ///
+    /// The update will be downloaded and staged, but will not be applied automatically on reboot.
+    /// Use `bootc upgrade --apply` to apply the staged update.
```

Comment:
How about

> Download the update and ensure it's retained on disk for the lifetime of this system boot, but it will not be applied on reboot. If the system is rebooted without applying the update, the image will be eligible for garbage collection again.